travel_platform/configurations.cfg

```diff
@@ -3,4 +3,14 @@ API_KEY = 9ba66b8d-2bdc-4674-bdc4-3cc67e2c25af
 API_URL = https://api.izi.travel
 API_VERSION = 1.2.4
 MEDIA_URL = https://media.izi.travel
-LANGUAGES = uk,en,ru
\ No newline at end of file
+LANGUAGES = uk,en,ru
+
+[api_skyscanner]
+api_key = ka775911102027919433182126616312
+market = 'UK'
+currency = 'GBP'
```

Comment:
what is the reason of all of these useless configs here? If in your examples you uses other values?